app/javascript/components/NavBar.vue

```diff
@@ -13,39 +13,51 @@
     </template>
 
     <template slot="end">
-      <b-navbar-item tag="div" v-if="visible('Feedback')">
-        <FeedbackButton action="software_feedbacks/new">Feedback</FeedbackButton>
-      </b-navbar-item>
       <b-navbar-item href="/glossary" v-if="visible('Glossary')">Glossary</b-navbar-item>
       <b-navbar-item href="/contributions" v-if="visible('Contributions')"
         >Contributions</b-navbar-item
       >
       <b-navbar-item href="/matches" v-if="visible('Matches')">Matches</b-navbar-item>
-      <b-navbar-item href="/admin" v-if="visible('Admin')">Admin</b-navbar-item>
       <b-navbar-item tag="div" v-if="visible('Login')">
         <a href="/users/sign_in" class="button is-outlined">Login</a>
       </b-navbar-item>
       <b-navbar-item tag="div" v-if="visible('Sign Up')">
         <a href="/users/sign_up" class="button is-outlined">Sign Up</a>
       </b-navbar-item>
-      <b-navbar-item tag="div" v-if="visible('Logout')">
-        <DeleteButton action="/users/sign_out">Logout</DeleteButton>
+      <b-navbar-item href="/admin" v-if="visible('Admin')">
+        Admin
       </b-navbar-item>
+
+      <b-navbar-dropdown label="Account" collapsible right v-if="visible('Logout')">
+        <div class="sub-item">
+          <a class="navbar-item" href="/users/edit">My profile</a>
+        </div>
+        <div class="sub-item">
+          <FeedbackButton
+            classes="navbar-item"
+            action="software_feedbacks/new"
```

Comment:
This `action` property seems to be obsolete (i know it existed before your changes -- must've been inadvertently left behind previously).